.circleci/config.yml

```diff
@@ -148,20 +128,18 @@ workflows:
   compile_lint_test_dist_deploy:
     jobs:
       - install-dependencies
-      - compile-libs:
+      - compile:
           requires: [install-dependencies]
       - lint:
           requires: [install-dependencies]
-      - dist-libs:
-          requires: [compile-libs]
-      - dist-apps:
-          requires: [compile-libs]
+      - dist:
+          requires: [compile]
       - test:
-          requires: [compile-libs]
+          requires: [compile]
       - deploy-preview:
-          requires: [dist-apps]
```

Comment:
since this only depended on `dist-apps` previously, it was never picking up the UMD bundles built in `dist-libs`